packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -202,56 +257,64 @@ export class WaferMap extends FoundationElement {
     }
 
     private quadrantChanged(): void {
-        this.queueRender();
-    }
-
-    private orientationChanged(): void {
-        this.queueRender();
+        this.waferMapUpdateTracker?.track('quadrant');
+        this.waferMapUpdateTracker?.queueUpdate();
     }
 
     private maxCharactersChanged(): void {
-        this.queueRender();
+        this.waferMapUpdateTracker?.track('maxCharacters');
```

Comment:
I am not very familiar with the `updateTracker`, but do we have to call the `track` method for each property/attribute change event ? This seems to be more like an initialization that has to be done just once ...

Reply:
when creating the tracker, we define which properties will be tracked. on every update we set which properties have changed, run the update specific to those changes and after we reset every tracked property. I will add some comments to make everything more clear